### PR TITLE
Fix style issues and clean up code

### PR DESCRIPTION
**Pull request by oliversalzburg**
This change fixes several issues with the code:

- Missing semicolons
- Double-quoted string
- Dead code
- Deprecated function calls
- JSDoc

### CODE REVIEW

**Comment by oliversalzburg** [comment]
There's more to do with this plugin, but this is a good start.

Ultimately, I want to remove `request` from it as well and give everything another review.

**Comment by oliversalzburg** [comment]
I know the linting fails, but it did that before the changes as well. I'll fix that in an upcoming patch.

File: lib/api.js [hunk 1]
```diff
@@ -307,13 +308,13 @@ function getRepos(token, username, callback) {
...
-        return callback(err)
+        console.error('get_github_repos() - Error fetching repos & orgs: %s', err);
+        return callback(err);
       }
-      if (!r) return callback('Get repos failed; no response')
+      if (!r) return callback('Get repos failed; no response');
 
       org_memberships = o.data;
-      for (var i=0; i<r.data.length; i++) {
+      for (var i = 0; i < r.data.length; i++) {
```
Comment (knownasilya) [inline lib/api.js]: this could be a `forEach`

Comment (oliversalzburg) [inline lib/api.js]: I'll take a look at it and update the PR. I'll probably make another pass through the whole code anyway and look for small improvements to make.

**Comment by oliversalzburg** [comment]
@knownasilya The `forEach` is implemented :)

**Pull request merged**